clang/test/Modules/compare-file-size.py

```diff
@@ -0,0 +1,22 @@
+import argparse
+import os
+
+def get_file_size(file_path):
+    try:
+        return os.path.getsize(file_path)
+    except:
+        print(f"Unable to get file size of {file_path}")
+        return None
```

Comment:
Is the `try/except` better than just letting the exception propagate up through main? I'm fine w/ it, but I wonder if `print` is better than the unhandled exception? it also doesn't fail, does it? So I don't think you'd return an error code if this didn't succeed, would you?

Reply:
Now that I look at it again, I think having an unhandled exception would be better. I didn't realize that my implementation doesn't actually return an error code, but I would think having the error code would be better for debugging.